tests/server/TestDiscoViews.js

```diff
@@ -20,9 +20,9 @@ describe('GET requests', () => {
     webpackIsomorphicTools.undo();
   });
 
-  for (const path of ['search', 'disco']) {
+  for (const path of ['/']) {
```

Comment:
Yeah I wasn't sure if we might want to add to it later. But certainly I can clean it up as we can easily add it back as needed.